app/packages/core/src/components/ColorModal/ColorFooter.tsx

```diff
@@ -54,24 +50,9 @@ const ColorFooter: React.FC = () => {
         >
           Reset
         </Button>
-        <Button
-          title={
-            canEdit
```

Comment:
This is the newer change and the one we want. Please see https://github.com/voxel51/fiftyone/pull/3573